app/views/devise/shared/_links.html.erb

```diff
@@ -3,7 +3,7 @@
 <% end %>
 
 <%- if devise_mapping.registerable? && controller_name != 'registrations' %>
-  <%= link_to "Sign up", new_registration_path(resource_name) %><br />
+  <%= link_to "Register", new_registration_path(resource_name) %><br />
 <% end %>
 
 <%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
```

Comment:
Its possible to change this logic for the conditional `devise_mapping.registerable? && controller_name != 'registrations'` rendering (and other conditionals that may repeat after) to a Helper file so it can be available to other views as well and can be reusable if needed, also it will be a little bit cleaner on the html part, what do you think?

Reply:
thanks for this observation, we will work in it right now!!

Reply:
DONE